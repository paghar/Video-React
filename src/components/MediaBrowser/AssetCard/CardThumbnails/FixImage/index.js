//@flow
import React from 'react';
import { ThumbnailImg } from './index.style';

type Props = {
  thumbnailSrc: string,
  className?: string,
};

const FixImageThumbnail = ({ className, thumbnailSrc }: Props) => {
  return <ThumbnailImg className={className} src={thumbnailSrc} />;
};

export default FixImageThumbnail;
