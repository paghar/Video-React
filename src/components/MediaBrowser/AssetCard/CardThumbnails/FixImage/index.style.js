//@flow
import styled from 'styled-components';

export const ThumbnailImg = styled.img`
  width: 100%;
  height: 100%;
  outline: none;
  display: flex;
  pointer-events: none;
  object-fit: contain;
`;
