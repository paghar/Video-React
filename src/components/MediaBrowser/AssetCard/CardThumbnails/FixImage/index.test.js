//@flow
import React from 'react';
import renderer from 'react-test-renderer';
import FixImage from '.';
import cardImage from 'assets/image/Colleagues_looking_at_information_on_mobile_phone_screen_Videos_Mixkit.png';

describe('<AssetCardFixImageThumbnail />', () => {
  it('Fix Image Thumbnail Snapshot test', () => {
    const tree = renderer
      .create(<FixImage thumbnailSrc={cardImage} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
