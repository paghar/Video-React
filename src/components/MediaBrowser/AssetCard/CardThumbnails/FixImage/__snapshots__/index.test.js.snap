// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssetCardFixImageThumbnail /> Fix Image Thumbnail Snapshot test 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  outline: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  pointer-events: none;
  object-fit: contain;
}

<img
  className="c0"
  src="Colleagues_looking_at_information_on_mobile_phone_screen_Videos_Mixkit.png"
/>
`;
